h2o-algos/src/main/java/hex/schemas/GBMV3.java

```diff
@@ -97,6 +99,11 @@ public static final class GBMParametersV3 extends SharedTreeV3.SharedTreeParamet
     @API(help="A set of allowed column interactions.", level= API.Level.expert)
     public String[][] interaction_constraints;
 
+    @API(help="In-training checkpoints", level = API.Level.expert, gridable = false)
+    public String in_training_checkpoints_dir;
+
+    @API(help="Checkpoint the model after every so many trees. Default is 1 when in_training_checkpoints_dir is not null.", level = API.Level.expert, gridable = false)
```

Comment:
The default is generated on the client-side - this makes it redundant - the python doc mentions it 2x.

Reply:
Thanks